templates/macros/_macros-highlights.jinja

```diff
@@ -0,0 +1,33 @@
+# Params
+# num_item: Number of items (requires 3-4 items)
+# Slots
+# highlights_item_content_[1-4]: Highlights list item content
+
+{%- macro highlights(num_items) -%}
+  <section class="p-strip">
+    <div class="row">
+      <p class="p-text--small-caps">Highlights</p>
+      <hr />
+    </div>
+    {% if num_items == 3 %}
```

Comment:
We are restricting to 3/4 highlight items only, I can add an `else` catch just in case. @juanruitina could you weigh in on how we'd want to handle the columns for unsupported user values?

Reply:
If there were more than 4 items, then it could turn into a regular list. But I don't know if we want to allow that (I don't have strong opinions against it. I defer to @lyubomir-popov on this.

Reply:
I think that's an elegant fallback